lighthouse-core/test/audits/byte-efficiency/byte-efficiency-audit-test.js

```diff
@@ -5,19 +5,17 @@
  */
 'use strict';
 
-const ByteEfficiencyAudit_ = require('../../../audits/byte-efficiency/byte-efficiency-audit.js');
-const NetworkNode = require('../../../lib/dependency-graph/network-node.js');
-const CPUNode = require('../../../lib/dependency-graph/cpu-node.js');
-const Simulator = require('../../../lib/dependency-graph/simulator/simulator.js');
-const PageDependencyGraph = require('../../../computed/page-dependency-graph.js');
-const LoadSimulator = require('../../../computed/load-simulator.js');
-
-
-const trace = require('../../fixtures/traces/progressive-app-m60.json');
-const devtoolsLog = require('../../fixtures/traces/progressive-app-m60.devtools.log.json');
-const traceM78 = require('../../fixtures/traces/lcp-m78.json');
-const devtoolsLogM78 = require('../../fixtures/traces/lcp-m78.devtools.log.json');
-const assert = require('assert').strict;
+import ByteEfficiencyAudit_ from '../../../audits/byte-efficiency/byte-efficiency-audit.js';
+import NetworkNode from '../../../lib/dependency-graph/network-node.js';
+import CPUNode from '../../../lib/dependency-graph/cpu-node.js';
+import Simulator from '../../../lib/dependency-graph/simulator/simulator.js';
+import PageDependencyGraph from '../../../computed/page-dependency-graph.js';
+import LoadSimulator from '../../../computed/load-simulator.js';
+import trace from '../../fixtures/traces/progressive-app-m60.json';
+import devtoolsLog from '../../fixtures/traces/progressive-app-m60.devtools.log.json';
```

Comment:
I was surprised to see json imports working here. I guess Jest makes some allowances for it.

Reply:
> I was surprised to see json imports working here. I guess Jest makes some allowances for it.

how is this working? We have the jest transforms turned off and node should only do the import with `--experimental-json-modules` turned on (and even then only as `import devtoolsLog from '../../fixtures/traces/progressive-app-m60.devtools.log.json' assert { type: 'json' };`)

Reply:
No idea! I was converting these to readJson, but once I noticed it I stopped doing that midway and just allowed these to stay as json imports. I should probably finish the job...

Reply:
> wdyt: should all these become readJson or should we stick to importing them since that (somehow) works?

Since the import is so simple I'm good with relying on whatever overly-magical thing jest is doing behind the scenes, with the hopes that by the time we need to do something different, import assertions will be stable in whatever minimum Node version we're using